src/main.c

```diff
@@ -113,7 +113,7 @@ int main(int argc, char **argv)
         fprintf(stderr, "Cannot load startup module: %s\n", startup_module_name);
         return EXIT_FAILURE;
     }
-    globalcontext_insert_module_with_filename(glb, mod, basename(startup_module_name));
+    globalcontext_insert_module_with_filename(glb, mod, basename((char *) startup_module_name));
```

Comment:
See #423 . `startup_module_name` variable really should be declared `char *` instead of `const char *`.